docs/basics/101-138-sharethirdparty.rst

```diff
@@ -0,0 +1,502 @@
+
+This means that someone who wants to access the data from dropbox needs to
+enable the special remote.
+For this,  this person first needs to install and configure ``rclone``
+as well: Since ``rclone`` is the protocol with which
+annexed data can be transferred from and to Dropbox, anyone who needs annexed
+data from Dropbox needs this special remote. Therefore, the first steps are
+the same as before:
+
+- `Install <https://rclone.org/install/>`__ ``rclone`` on your system.
+- Run ``rclone config`` to configure ``rclone`` to work with Dropbox. It is
+  important to name the remote "dropbox-remote" (i.e., give it the same name
+  as the one configured in the dataset).
+- ``git clone`` the
+  `git-annex-remote-rclone <https://github.com/DanielDent/git-annex-remote-rclone>`_
+  repository to your machine.
+- Copy the path to this repository into your ``$PATH`` variable. If the
+  clone is in ``/home/user-bob/repos``, the command would look like this [#f4]_::
+
+  $ export PATH="/home/user-bob/repos/git-annex-remote-rclone:$PATH"
+
+After this is done, you can execute what DataLad's output message suggests
+to "enable" this special remote (inside of the installed ``DataLad-101``)::
+
+   $ datalad siblings -d "/Users/awagner/Documents/DataLad-101" enable -s dropbox-remote
+   .: dropbox-remote(?) [git]
+
+And once this is done, you can get any annexed file contents, for example the
+books, or the cropped logos from chapter "DataLad, Run!"::
+
+   $ datalad get books/TLCL.pdf
+   get(ok): /home/some/other/user/DataLad-101/books/TLCL.pdf (file) [from dropbox-remote]
+
+Built-in data export
+^^^^^^^^^^^^^^^^^^^^
+
+Apart from flexibly configurable special remotes that allow publishing
+annexed content to a variety of third party infrastructure, DataLad also has
+some build-in support for "exporting" data to other services.
+
+One example is the command :command:`export-to-figshare`. Running
+this command allows you to publish the dataset to
+`Figshare <https://figshare.com/>`_. The main difference is
+that this moves data out of version control and decentralized tracking,
+and essentially "throws it over the wall". This means, while your data (also
+the annexed data) will be available for download on Figshare, none of the
+special features a DataLad dataset provides will be available, such as its
+history or configurations.
+
+.. rubric:: Footnotes
+
+.. [#f1] :term:`GitLab`, on the other hand, provides a git-annex configuration. It
+         is disabled by default, and to enable it you would need to have administrative
+         access to the server and client side of your GitLab instance. Find out more
+         `here <https://docs.gitlab.com/ee/administration/git_annex.html>`_.
+
+.. [#f2] If you have an unused computer and a bit of time to spare, here is an
+         instruction `how to build your own webserver <https://www.instructables.com/id/Set-up-your-very-own-Web-server/>`_.
```

Comment:
While technically simple, I would be reluctant to advise people who have no experience in running web infrastructure to do so after reading an online post. Soon this thing will be a spam bot and we will all have to suffer from it ;-)